### DISCUSSION TITLE
The webpage deploys correctly and visible on gh-pages, but becomes blank if try to transfer in to another domain name.

### DISCUSSION THREAD

**Discussion by MolchanovaA** [community/community#103142] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

hi! 
I have bought a domain name with name cheap, did all instucrutions to redirect my webpage from ghpages to new website name. 
Support said that I have done all setting well, from their side so issues so far. 
Before I have deployed it on gh pages, no errors, no issues, looks nicely and working. 

But when I try to change 'Custum domain' on pages to my website name, it redirects me to the empty white page.
Good thing that if I oped devtools, I can see my site structure, <head> has all imports, but <body> empty.
It gives me error: 
Failed to load resource: the server responded with a status of 404 ()
repo: https://github.com/MolchanovaA/namysto
new site name: namysto.org

**Comment by queenofcorgis** [comment]
Do you have a CNAME file in the repo?

**Reply by MolchanovaA** [reply]
no, I don't have any likes with such mane. what is it? what has to be inside?
